cargo-c defaults to static on musl, we want dynamic.

Source: https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/gst-plugins-rs/0002-dylib.patch

diff --git a/cargo_wrapper.py b/cargo_wrapper.py
index 641edc8..4859b6b 100644
--- a/cargo_wrapper.py
+++ b/cargo_wrapper.py
@@ -111,6 +111,7 @@ if __name__ == "__main__":
             cargo_cmd += ['build']
         else:
             cargo_cmd += ['cbuild']
+            cargo_cmd += ['--library-type=cdylib']
             if not opts.disable_doc:
                 features += ['doc']
         if opts.target == 'release':
